src/web/mjs/connectors/MangaDisk.mjs

```diff
@@ -0,0 +1,37 @@
+import WordPressMangastream from './templates/WordPressMangastream.mjs';
+export default class MangaDisk extends WordPressMangastream {
+
+    constructor() {
+        super();
+        super.id = 'mangadisk';
+        super.label = 'Manga Disk';
+        this.tags = [ 'manga', 'indonesian' ];
+        this.url = 'https://mangadisk.web.id';
+        this.path = '/manga/list-mode/';
+        this.queryPages = 'div#readerarea canvas';
+    }
+
+    async _getPages(chapter) {
+        const uri = new URL(chapter.id, this.url);
+        let request = new Request(uri, this.requestOptions);
+        let data = await this.fetchDOM(request, this.queryPages);
+
+        return data.map(hash => {
+            const payload = {
+                url: 'https://img.nesia.my.id/image?id='+hash.attributes["file-id"].value,
+                referer: this.url
+            };
+            return this.createConnectorURI(payload);
+        });
+    }
+
+    async _handleConnectorURI(payload) {
+        let request = new Request(new URL(payload.url), this.requestOptions);
+        request.headers.set('x-referer', payload.referer, this.url);
```

Comment:
The `Headers.set()` method only supports two parameters, the 3rd will be ignored